src/keycloak/chart/templates/uds-package.yaml

```
@@ -43,6 +43,15 @@ spec:
           - 80
         remoteGenerate: Anywhere
 
+      {{- if .Values.smtp.enabled }}
+        - description: "SMTP access"
+          direction: Egress
+          selector:
+            app.kubernetes.io/name: keycloak
+          port: {{ .Values.smtp.port }}
+          remoteGenerate: Anywhere
```

Indentation here is off compared to the netpol below/above.

```suggestion
      - description: "SMTP access"
        direction: Egress
        selector:
          app.kubernetes.io/name: keycloak
        port: {{ .Values.smtp.port }}
        remoteGenerate: Anywhere
```